src/modules/todo-List.js

```diff
@@ -0,0 +1,115 @@
+import DynamicTodo from './dynamic-Todo';
+import CreateTodo from './create-Todo';
+import TodoStatus from './todo-Status';
+import { rotate, rotateReverse } from './rotate-RefreshBtn';
+
+const dynamicTodo = new DynamicTodo();
+
+class TodoList {
+  constructor() {
+    this.todoList = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  // Model
+  addTodo(todo) {
+    this.todoList.push(todo);
+    this.saveTodos();
+    rotate();
+    dynamicTodo.render(this.todoList, this);
+  }
+
+  removeTodo(todoId) {
+    this.todoList = this.todoList.filter(({ id }) => id !== todoId);
+    this.reindexTodos();
+    this.saveTodos();
+    rotateReverse();
+  }
+
+  // Sets Todo for Editing
+  setEditing(todoToEdit) {
+    this.todoList[todoToEdit - 1].isEditing = true;
+    this.saveTodos();
+  }
+
+  // Updates a Todo
+  updateTodo(id, description) {
+    this.todoList[id].description = description;
+    this.todoList[id].isEditing = false;
+    this.saveTodos();
+  }
+
+  onEdit(todoId) {
+    this.setEditing(todoId);
+    dynamicTodo.render(this.todoList, this);
```

Comment:
Consider implementing a function to reset whenever a user try to edit another task without clicking the update button on the previous task being edited.

The will give the user assurance that his/her changes have been saved.

Reply:
I would like it if you made this point clear, having the Update button alerts the user that he is editing something.